### DISCUSSION TITLE
eror saat interpretasi lime

### DISCUSSION THREAD

**Discussion by hairulysin** [teamalgoritma/community#732] [Q&A] [answered]
Saya coba melakukan interpretasi pada model Random Forest menggunakan metode LIME. . Namun, saya terus mendapatkan error berikut: "Error: The class of model must have a model_type method. See ?model_type to get an overview of models supported out of the box". Saya telah membuat fungsi model_type untuk menetapkan tipe model "classification", namun tetap eror yang sama.

**Comment by divakartika** [comment, marked as answer]
Dear @hairulysin 

Boleh dipastikan apakah pembuatan model Random Forest menggunakan fungsi `randomForest()`? Jika demikian, maka bisa dicoba untuk membuat model Random Forest menggunakan fungsi `train()` dengan parameter `method = "rf"` (seperti pembelajaran di kelas). Hal ini dikarenakan library LIME belum kompatibel dengan hasil fungsi `randomForest()`. Semoga membantu :)

Regards,
Team Algoritma

**Comment by kevbow** [comment]
Halo @hairulysin, untuk menjaga agar forum diskusi tetap berjalan kondusif, kami izin melakukan lock/close diskusi ini. Semoga dapat dipahami.
